python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -660,6 +673,7 @@ def _aggs_supported(arg, supported: set):
     return False
 
 
+@_dask_cudf_nvtx_annotate
```

Comment:
Noting that I've added this decorator as it seems that the other related internal helper functions have been decorated